viniciusscala/PTA-challenge3
# Feature/complete project

## What I did
- Finish the website structure and style
- Make it responsive
- Connect the front-end to the back-end
- Add search feature

## How to test
1. Clone the repository
2. Open 2 terminals in the folder of this project
3. cd /server
4. node/nodemon index.js
5. In the second terminal, cd /client
6. yarn/npm start
7. Go to http://localhost:3000/
8. Try changing your browser's window size to test if it is responsive
9. Type something in the search bar
10. check if the list only show itens that match what you typed
